examples/app-autotune.yaml

```diff
@@ -5,7 +5,8 @@ metadata:
   namespace: "default"
 spec:
   slo:
-    objective_function: "transaction_response_time"
+    objective_function:
+      - type: source
```

Comment:
Change this to expression and keep `transaction_response_time`

Reply:
Done